.pre-commit-config.yaml

```diff
@@ -22,6 +22,13 @@ repos:
       - id: python-check-blanket-type-ignore
       - id: python-check-mock-methods
 
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v2.29.0
+    hooks:
+      - id: pyupgrade
+        args: [--py36-plus]
+        exclude: '^(install|get)-poetry.py$'
```

Comment:
These files are excluded as `get-poetry.py` is in maintenance mode and `install-poetry.py` is vendored from a different repo.